<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerMenu</class>
 <widget class="QMainWindow" name="CustomerMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(214, 231, 243);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
	font-size:32px;
}
</string>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
	font-size:32px;
}
</string>
    </property>
    <property name="text">
     <string>email</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="allItems">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>451</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget
 {
    background-color: white;
    border-radius: 10px;
}

QTableView 
{
    border: none;
}


QScrollBar:vertical {
    background-color: #FFFFFF;
    border-radius: 10px;
	margin-right: 5px;
}

QScrollBar::handle:vertical {
    background-color: #4DA6FF;
    border-radius: 10px;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
}

QTableWidget::item:selected {
    background-color: #4DA6FF;
    color: white;
}

QTableWidget::item:selected:focus {
    outline: none;
}

QTableWidget::item:selected:active {
    border-top-left-radius: 12px;
    border-bottom-left-radius: 12px;
    border-top-right-radius: 12px;
    border-bottom-right-radius: 12px;
}

QTableWidget::item:selected:first {
    border-top-left-radius: 12px;
    border-bottom-left-radius: 12px;
}

QTableWidget::item:selected:last {
    border-top-right-radius: 12px;
    border-bottom-right-radius: 12px;
}
</string>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Catagory</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="itemCart">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>551</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget
 {
    background-color: white;
    border-radius: 10px;
}

QTableView 
{
    border: none;
}


QScrollBar:vertical {
    background-color: #FFFFFF;
    border-radius: 10px;
	margin-right: 5px;
}

QScrollBar::handle:vertical {
    background-color: #4DA6FF;
    border-radius: 10px;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
}

QTableWidget::item:selected {
    background-color: #4DA6FF;
    color: white;
}

QTableWidget::item:selected:focus {
    outline: none;
}

QTableWidget::item:selected:active {
    border-top-left-radius: 5px;
    border-bottom-left-radius: 5px;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}

QTableWidget::item:selected:first {
    border-top-left-radius: 5px;
    border-bottom-left-radius: 5px;
}

QTableWidget::item:selected:last {
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
</string>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>item No.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Catagory</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>All Items</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>Cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doneShoping">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>570</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2196F3;
    color: #FFFFFF;
    border-radius: 20px;
    padding: 10px 20px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1976D2;
}

QPushButton:pressed {
    background-color: #0D47A1;
}
</string>
    </property>
    <property name="text">
     <string>Done Shoping</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addToCart">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2196F3;
    color: #FFFFFF;
    border-radius: 14px;
    padding: 4px 10px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1976D2;
}

QPushButton:pressed {
    background-color: #0D47A1;
}
</string>
    </property>
    <property name="text">
     <string>Add to Cart</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="itemQuantity">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>560</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
                           background-color: rgb(85, 170, 255);
                           color: #FFFFFF;
                           border-radius: 15px;
                           padding: 8px;
		font-size: 14px;
                           font-weight: bold;
}
 QSpinBox::up-button, QSpinBox::down-button 
{
	background-color: white;
	border-radius: 6px;
	padding: 5px;
	padding-right: 10px;
}
 QSpinBox::up-button:hover, QSpinBox::down-button:hover 
{
	background-color: #0D47A1;
	padding-right: 10px;
 }
 QSpinBox::up-button:pressed, QSpinBox::down-button:pressed 
{
	background-color: #0D47A1;
	padding-right: 10px;
 }</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selectedItem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border-radius: 15px;
    background-color: white;
	padding-left: 10px;
    padding-right: 10px; 
	font-size: 14px;
 	color: #4da6ff;
    font-weight: bold;
	text-align: center;
}

QLineEdit:focus, QLineEdit:hover {
    border: 1px solid #4da6ff;
}</string>
    </property>
    <property name="text">
     <string>Selected Item</string>
    </property>
   </widget>
   <widget class="QComboBox" name="itemCatagory">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>560</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #ffffff;
    border-radius: 15px;
    padding: 5px;
    color: #4da6ff;
    font-size: 14px;
    font-weight: bold;
    text-align: center;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left-width: 0px;
    border-top-right-radius: 20px;
    border-bottom-right-radius: 20px;
}

QComboBox::down-arrow {
    image: url(icons/down-arrow.png);
    width: 16px;
    height: 16px;
}

QComboBox QAbstractItemView {
    selection-background-color: transparent;
    selection-color: white;
    background-color: white;
    border-radius: 15px;
    padding: 5px;
    color: black;
    border: none;
    outline: none;
    scrollbar-color: #CCCCCC #F5F5F5;
    font-weight: regular;
}

QComboBox QAbstractItemView::item {
    height: 30px;
}

QComboBox QAbstractItemView::item:hover {
    background-color: white;
    border-radius: 15px;
    font-weight: bold;
}

QComboBox QAbstractItemView::item:selected {
    border-radius: 15px;
    color: #4da6ff;
}

QComboBox QAbstractItemView::item:focus {
    background-color: transparent;
    font-weight: bold;
}

QComboBox::focus {
    border: none;
}

QComboBox QAbstractItemView::scrollbar {
    background-color: #F5F5F5;
    width: 10px;
    border-radius: 5px;
}

QComboBox QAbstractItemView::scrollbar:vertical {
    margin-right: 5px;
}

QComboBox QAbstractItemView::scrollbar-handle:vertical {
    background-color: #CCCCCC;
    border-radius: 5px;
    min-height: 20px;
}

QComboBox QAbstractItemView::scrollbar-handle:vertical:hover {
    background-color: #BBBBBB;
}

QComboBox QAbstractItemView::scrollbar:horizontal {
    margin-bottom: 5px;
}

QComboBox QAbstractItemView::scrollbar-handle:horizontal {
    min-width: 20px;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selectedItem_Cart">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>380</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border-radius: 15px;
    background-color: white;
	padding-left: 10px;
    padding-right: 10px; 
	font-size: 14px;
 	color: #4da6ff;
    font-weight: bold;
	text-align: center;
}

QLineEdit:focus, QLineEdit:hover {
    border: 1px solid #4da6ff;
}</string>
    </property>
    <property name="text">
     <string>Selected Item</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="itemQuantity_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>380</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
                           background-color: rgb(85, 170, 255);
                           color: #FFFFFF;
                           border-radius: 15px;
                           padding: 8px;
		font-size: 14px;
                           font-weight: bold;
}
 QSpinBox::up-button, QSpinBox::down-button 
{
	background-color: white;
	border-radius: 6px;
	padding: 5px;
	padding-right: 10px;
}
 QSpinBox::up-button:hover, QSpinBox::down-button:hover 
{
	background-color: #0D47A1;
	padding-right: 10px;
 }
 QSpinBox::up-button:pressed, QSpinBox::down-button:pressed 
{
	background-color: #0D47A1;
	padding-right: 10px;
 }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFromCart">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>380</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2196F3;
    color: #FFFFFF;
    border-radius: 14px;
    padding: 4px 10px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1976D2;
}

QPushButton:pressed {
    background-color: #0D47A1;
}
</string>
    </property>
    <property name="text">
     <string>Remove From Cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>380</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2196F3;
    color: #FFFFFF;
    border-radius: 14px;
    padding: 4px 10px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1976D2;
}

QPushButton:pressed {
    background-color: #0D47A1;
}
</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>440</y>
      <width>52</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
	font-size:16px;
}
</string>
    </property>
    <property name="text">
     <string>Cash</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
	font-size:16px;
}
</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cash">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>440</y>
      <width>166</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border-radius: 15px;
    background-color: white;
	padding-left: 10px;
    padding-right: 10px; 
	font-size: 14px;
 	color: #4da6ff;
    font-weight: bold;
	text-align: center;
}

QLineEdit:focus, QLineEdit:hover {
    border: 1px solid #4da6ff;
}</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="change">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>480</y>
      <width>166</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border-radius: 15px;
    background-color: white;
	padding-left: 10px;
    padding-right: 10px; 
	font-size: 14px;
 	color: #4da6ff;
    font-weight: bold;
	text-align: center;
}

QLineEdit:focus, QLineEdit:hover {
    border: 1px solid #4da6ff;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalItems">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>470</y>
      <width>74</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border-radius: 15px;
    background-color: white;
	padding-left: 10px;
    padding-right: 10px; 
	font-size: 14px;
 	color: #4da6ff;
    font-weight: bold;
	text-align: center;
}

QLineEdit:focus, QLineEdit:hover {
    border: 1px solid #4da6ff;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalPrice">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>430</y>
      <width>74</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border-radius: 15px;
    background-color: white;
	padding-left: 10px;
    padding-right: 10px; 
	font-size: 14px;
 	color: #4da6ff;
    font-weight: bold;
	text-align: center;
}

QLineEdit:focus, QLineEdit:hover {
    border: 1px solid #4da6ff;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="discount">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>510</y>
      <width>74</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border-radius: 15px;
    background-color: white;
	padding-left: 10px;
    padding-right: 10px; 
	font-size: 14px;
 	color: #4da6ff;
    font-weight: bold;
	text-align: center;
}

QLineEdit:focus, QLineEdit:hover {
    border: 1px solid #4da6ff;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
	font-size:16px;
}
</string>
    </property>
    <property name="text">
     <string>Total Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>84</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
	font-size:16px;
}
</string>
    </property>
    <property name="text">
     <string>Total Items</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>510</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
	font-size:16px;
}
</string>
    </property>
    <property name="text">
     <string>Discount</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>507</x>
      <y>544</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="logOut">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2196F3;
    color: #FFFFFF;
    border-radius: 14px;
    padding: 4px 10px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1976D2;
}

QPushButton:pressed {
    background-color: #0D47A1;
}
</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
